hikari/impl/cache.py

```diff
@@ -165,9 +165,11 @@ def set_dm_channel_id(
         channel: snowflakes.SnowflakeishOr[channels.PartialChannel],
         /,
     ) -> None:
-        user = snowflakes.Snowflake(user)
-        if self._is_cache_enabled_for(config.CacheComponents.EMOJIS) and user in self._user_entries:
-            self._dm_channel_entries[user] = snowflakes.Snowflake(channel)
+        user_id = snowflakes.Snowflake(user)
+        # XXX: Should this require the full user and cache that if not there?
```

Comment:
Rather than adding in-code TODOs can we migrate these to a github project where they can actually be tracked?

Reply:
Sure. This one in particular was to open a discussion and not be merged till removed (flake8 will complain about XXX but not TODO's)

Reply:
Removed the XXX and changed the code a bit :+1: